inst/extdata/whitespace-sample.txt

```diff
@@ -0,0 +1,4 @@
+firstName      lastName       state        phoneNumber
+John   Smith                WA        418-Y11-4111
+Mary               Hartford       CA         319-Z19-4341
+Evan Nolan          IL         219-532-c301
```

Comment:
This is on the right track.

I propose using column names that don't raise the question of camelCase vs. snake_case: use something more like `first`, `last`, `state`, `phone`.

Then I would also just use 1 space everywhere, maybe? Right now, the whitespace feels sort of random. It's not constant (e.g. 1 space) but it's also not aligned in any obvious way.

Reply:
I created the file this way to "show off" what `read_table()` could do. I liked that when I read it in with `read_fwf()` there were empty cells but with `read_table()` it read it in cleanly.

I can make a file that isn't as scattered but that still does the above if you think that'd be better.